index.html

```diff
@@ -47,6 +49,97 @@ <h3 class="octopus__subtitle">
+                <span class="diver__card-number">02</span>
+                На Вашем сайте появится кнопка и всплывающее окно
+              </dt>
+              <dd class="diver__card-description">
+                Посетители заходят на Ваш сайт, видят кнопку или всплывающее
+                окно и вводят свой телефон
+              </dd>
+            </div>
+            <div class="diver__card">
+              <dt class="diver__card-termin">
+                <span class="diver__card-number">03</span>
+                Сервис автоматически звонит менеджеру и клиенту
+              </dt>
+              <dd class="diver__card-description">
+                Как только менеджер взял трубку, сервис автоматически
+                связывается с клиентом и соединяет его с менеджером. В среднем
+                за 18,5 сек
+              </dd>
+            </div>
+            <div class="diver__card">
+              <dt class="diver__card-termin">
+                <span class="diver__card-number">04</span>
+                Клиент теперь Ваш !
+              </dt>
+              <dd class="diver__card-description">
+                Уже через 28 секунд Ваш сотрудник общается с клиентом и делает
+                продажу!
+              </dd>
+            </div>
+          </dl>
+        </div>
+        <div class="diver__video">
+          <div class="video__container container">
+            <h2 class="diver__text">
+              <span>Посмотрите видео</span>
+              <span>как работает виджет</span> 
+            </h2>
+          </div>
+          <iframe
+            class="video"
+            width="100%"
+            height="668px"
+            src="https://www.youtube.com/embed/KQKCLd_6DGw"
+            title="YouTube video player"
+            frameborder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullscreen
+          >
+          </iframe>
+        </div>
+      </section>
     </main>
+    <footer class="footer">
+      <section class="footer__container container">
+        <div class="footer__tariffs">
+          <article class="footer__tariff footer__tariff-left">
+            <h2 class="footer__short-title">
+              <span>Бесплатно</span>
+              <span>О руб.</span>
+            </h2>
+            <p class="footer__text">
+              Тестовая версия на 14 дней и 20 руб. на балансе для проверки
+              качества связи
+            </p>
+            <button type="button" class="footer__button">ПОПРОБОВАТЬ</button>
+          </article>
+          <article class="footer__tariff footer__tariff-right">
+            <h2 class="footer__short-title">При оплате на 1 год от 299 руб.</h2>
+            <p class="footer__text">
+              Платите только за наш сервис, связь оплачивается по минимальной
+              цене 4 руб. за минуту.
+            </p>
+            <button type="button" class="footer__button">ПОДКЛЮЧИТЬ</button>
```

Comment:
```suggestion
            <button type="button" class="footer__button button-outlined button-outlined_md">ПОДКЛЮЧИТЬ</button>
```
Add modification to styles.css
```
 .button-outlined_md {
    font-size: 36px;
    line-height: 42px;
 }
```
**Please rename `button-outline` in `style.css` to `button-outlined`.**